portfolio/src/main/webapp/script.js

```diff
@@ -13,19 +13,70 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
 function addRandomFact() {
-  const facts =
-      ["I\'ve lived in LA my whole life", 
-       "I have surpassed level 3000 on Candy Crush Saga", 
-       "I have an older sister", 
-       "My favorite fruit is watermelon"];
-
-  // Pick a random fact.
-  const fact = facts[Math.floor(Math.random() * facts.length)];
-
-  // Add it to the page.
-  const factContainer = document.getElementById('fact-container');
-  factContainer.innerText = fact;
+    // List of facts to choose from.
+    const facts =
+        ["I've lived in LA my whole life", 
+        "I have surpassed level 3000 on Candy Crush Saga", 
+        "I have an older sister", 
+        "My favorite fruit is watermelon"];
+
+    const fact = facts[Math.floor(Math.random() * facts.length)];
+
+    // Add it to the page.
+    const factContainer = document.getElementById('fact-container');
+    factContainer.innerText = fact;
+}
+
+/**
+ * Load a specified number of comments to the comments section.
+ * If the specified number is greater than the total number of comments, all
+ * comments are shown.
+ */
+function loadComments() {
+    var numCommentsShown = parseInt(document.getElementById('comment-choice').value);
+
+    fetch("/data?limit=".concat(numCommentsShown))
+    .then(response => response.json())
+    .then((messages) => {
+        const messageListElement = document.getElementById('comments-container');
+        messageListElement.innerHTML = "";
+        messages.forEach((comment) => {
+            messageListElement.appendChild(createCommentElement(comment));
+        });
+    });
+}
+
+/**
+ * Create comment on the page with a delete button.
+ * After a comment is deleted, the comments are reloaded with the correct 
+ * number of comments shown.
+ */
+function createCommentElement(comment) {
+    const coElement = document.createElement('li');
```

Comment:
The variable name like 'coElement' is unusual for JavaScript. How about 'commentElement'?

Reply:
fixed!